src/components/layout/Layout.vue

```diff
@@ -4,6 +4,7 @@
     <sidebar></sidebar>
     <div class="content-wrap" id="content-wrap">
       <main id="content" class="content" role="main">
+        <vuestic-breadcrumbs></vuestic-breadcrumbs>
```

Comment:
On Dasboard page breadcrumbs and green alert look no good together. Please, rm green alert from dashboard and it's text from i18n files